frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -14,9 +17,7 @@ pods:
           type: ROOT
           size: {{BROKER_DISK}}
         goal: RUNNING
-        cmd: "env && exec $MESOS_SANDBOX/kafka_2.11-0.10.0.0/bin/kafka-server-start.sh $MESOS_SANDBOX/kafka_2.11-0.10.0.0/config/server.properties"
-        uris:
-          - "https://downloads.mesosphere.com/kafka/assets/kafka_2.11-0.10.0.0.tgz"
+        cmd: "./bootstrap -resolve=false && exec $MESOS_SANDBOX/kafka_2.11-0.10.0.0/bin/kafka-server-start.sh $MESOS_SANDBOX/kafka_2.11-0.10.0.0/config/server.properties"
```

Comment:
YAML supports multline strings like this:
https://github.com/mesosphere/dcos-commons/blob/master/frameworks/helloworld/src/main/dist/svc.yml#L31-L34

Also we're very inconsistent throughout our YAML files.  Generally speaking all of our strings do not need to use `"` marks, so if you could remove them when you see them, that'd be cool.